libs/SalesforceAnalytics/src/com/salesforce/androidsdk/analytics/store/EventStoreManager.java

```diff
@@ -158,16 +156,9 @@ public InstrumentationEvent fetchEvent(String eventId) {
      * Deletes a specific event stored on the filesystem.
      *
      * @param eventId Unique identifier for the event.
-     * @return True - if successful, False - otherwise.
      */
-    public boolean deleteEvent(String eventId) {
-        if (TextUtils.isEmpty(eventId)) {
-            SalesforceAnalyticsLogger.e(context, TAG, "Invalid event ID supplied: " + eventId);
-            return false;
-        }
-        final String filename = eventId + filenameSuffix;
-        final File file = new File(rootDir, filename);
-        return file.delete();
+    public void deleteEvent(String eventId) {
```

Comment:
SemVer?  You could `return !book.contains(eventId);` but I doubt anyone will ever care.  Probably not worth fixing.

Reply:
Fixed in #2036